network/manager.go

```diff
@@ -827,7 +842,9 @@ func cnsEndpointInfotoCNIEpInfos(endpointInfo restserver.EndpointInfo, endpointI
 		epInfo.HNSNetworkID = ipInfo.HnsNetworkID
 		epInfo.MacAddress = net.HardwareAddr(ipInfo.MacAddress)
 		epInfo.NetworkContainerID = ipInfo.NetworkContainerID
-
+		if epInfo.NetNsPath == "" {
+			epInfo.NetNsPath = netns
```

Comment:
can you explain this? add to the comment-- the "netns" variable is more like a defaultNetNs, which only populates if epInfo.NetNsPath is empty. Why do we add this (whereas we didn't need it before) and where is it used? Only place I can immediately see is in secondary endpoitn client if there is a cns connection failure and we perform async delete-- but generateEndpointLocally alrady takes the netns from args.Netns ?